tap_github/streams.py

```diff
@@ -130,28 +130,131 @@ def http_headers(self) -> dict:
 
     schema = th.PropertiesList(
         th.Property("id", th.IntegerType),
+        th.Property("node_id", th.StringType),
+        th.Property("url", th.StringType),
+        th.Property("html_url", th.StringType),
         th.Property("repo", th.StringType),
         th.Property("org", th.StringType),
-        th.Property("issue_number", th.IntegerType),
+        th.Property("number", th.IntegerType),
         th.Property("updated_at", th.DateTimeType),
         th.Property("created_at", th.DateTimeType),
-        # th.Property("closed_at", th.DateTimeType),  # Nulls causing parse error
+        th.Property("closed_at", th.DateTimeType),
```

Comment:
Thanks for restoring this. I think the null behavior problem came up very early in testing and I forgot to retest and restore the field.